colaraz/cms/templates/widgets/header.html

```diff
@@ -204,15 +204,16 @@ <h2 class="sr-only">${_("Account Navigation")}</h2>
   </header>
     % if user.is_authenticated:
       <%
-        colaraz_ecosystem_url = '{}://{}.{}/cvsocial/activity?{}'.format(protocol, identifier, domain, user_urls.get('ecosystem')) if user_urls.has_key('ecosystem') else '#'
-        colaraz_recruiting_url = '{}://{}.{}/{}'.format(protocol, identifier, domain, user_urls.get('recruiting')) if user_urls.has_key('recruiting') else '#'
-        colaraz_management_url = '{}://{}.{}/{}'.format(protocol, identifier, domain, user_urls.get('management')) if user_urls.has_key('management') else '#'
-        colaraz_performance_url = '{}://{}.{}/{}'.format(protocol, identifier, domain, user_urls.get('performance')) if user_urls.has_key('performance') else '#'
-        colaraz_development_url = '{}://{}.{}/{}'.format(protocol, identifier, domain, user_urls.get('development')) if user_urls.has_key('development') else '#'
-        colaraz_settings_url = '{}://{}.{}/{}'.format(protocol, identifier, domain, user_urls.get('settings')) if user_urls.has_key('settings') else '#'
-        colaraz_recommendations_url = '{}://{}.{}/{}'.format(protocol, identifier, domain, user_urls.get('recommendations')) if user_urls.has_key('recommendations') else '#'
-        colaraz_mentoring_url = '{}://{}.{}/{}'.format(protocol, identifier, domain, user_urls.get('mentoring')) if user_urls.has_key('mentoring') else '#'
-        colaraz_scholarship_url = '{}://{}.{}/{}'.format(protocol, identifier, domain, user_urls.get('scholarship')) if user_urls.has_key('scholarship') else '#'
+            colaraz_ecosystem_url = '{}cvsocial/activity?{}'.format(base_url, user_urls.get('ecosystem')) if user_urls.has_key('ecosystem') else '#'
```

Comment:
nit:
As we are refactoring code, can we replace `.has_key()` method with `in dict` syntax? as `.has_key()` is not supported in python3.